Packages/manifest.json

```diff
@@ -19,6 +19,7 @@
     "com.unity.test-framework": "1.1.29",
     "com.unity.textmeshpro": "3.0.6",
     "com.unity.timeline": "1.4.8",
+    "com.unity.toolchain.win-x86_64-linux-x86_64": "0.1.21-preview",
```

Comment:
Curious where this came from. Was it to test a linux device?

Reply:
I'm not sure--I included a module for building for linux to my editor install just in case, so that could be it. Should I just revert this file?

Reply:
And I guess the packages-lock file as well?

Reply:
Yes. Just going off of our Readme, we don't officially support it yet. So I would remove it for now until we officially do.

Reply:
Word, will do!